database/schedule/opts.go

```diff
@@ -22,6 +22,16 @@ func WithClient(client *gorm.DB) EngineOpt {
 	}
 }
 
+// WithEncryptionKey sets the encryption key in the database engine for Repos.
+func WithEncryptionKey(key string) EngineOpt {
+	return func(e *engine) error {
+		// set the encryption key in the repo engine
+		e.config.EncryptionKey = key
```

Comment:
```suggestion
// WithEncryptionKey sets the encryption key in the database engine for Schedules.
func WithEncryptionKey(key string) EngineOpt {
	return func(e *engine) error {
		// set the encryption key in the schedule engine
		e.config.EncryptionKey = key
```

looks like a previous oversight here as well: https://github.com/go-vela/server/blob/8ad123451469039e0125af1943e916c211415642/database/build/opts.go#L25-L28

Reply:
Good catches